### DISCUSSION TITLE
HyperTuning AttributeError: 'dict' object has no attribute '__name__'

### DISCUSSION THREAD

**Discussion by VladHardAndSoftWare** [RUCAIBox/RecBole#1757] [Q&A]
I am using hyperopt 0.2.5, with model recbole I mention my model and model settings in config.yaml file and I got `AttributeError: 'dict' object has no attribute '__name__'.`

![image](https://user-images.githubusercontent.com/42782084/236000291-922445cf-5e56-4fd5-93cb-b8c890fd82c2.png)
![image](https://user-images.githubusercontent.com/42782084/236000450-228c9ee0-77f9-407f-a4ff-2943a62aa619.png)

**Comment by Sherry-XLL** [comment]
Hello @VladHardAndSoftWare, thanks for your attention to RecBole!

You just need to comment out the dictionary after the `model` and assign values to the configuration as follows.

```
train_neg_sample_args: None
loss_type: CE
```

If you have further questions, feel free to contact us in this discussion. Also, an [issue](https://github.com/RUCAIBox/RecBole/issues) is more recommended if you have questions about the usage of RecBole.

**Reply by VladHardAndSoftWare** [reply]
Thank you for your answer. But, when I do this, I have ValueError: train_neg_sample_args [None] should be None when the loss_type is CE. 

![image](https://user-images.githubusercontent.com/42782084/236190906-2168bd6f-1462-420a-92c4-d93a6ebd7656.png)
![image](https://user-images.githubusercontent.com/42782084/236190835-b7dce5a6-9009-4126-a8f4-b2d41d155de6.png)
